sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs

```diff
@@ -49,7 +49,7 @@ public class ServiceBusAdministrationClient
         private const string RulesPath = "{0}/Subscriptions/{1}/rules";
 
         /// <summary>
-        /// Parameterless constructor to allow mocking.
+        /// Parameter-less constructor to allow mocking.
```

Comment:
Why this change?

Reply:
I'll revert those with this change. It doesn't exist in the dictionary used for validation, but it clearly understood as a word at this point in computer science.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-constructors

Reply:
😄    I wasn't going to pick on it, and it is grammatically correct, but I really didn't like the hyphen.

Reply:
```suggestion
        /// Parameterless constructor to allow mocking.
```